src/endpoint.ts

```diff
@@ -51,8 +51,10 @@ export class Endpoint {
     return { arrayFormat: 'brackets' }
   }
 
-  get paramsSerializer(): (params: any) => string {
-    return (params): string => qs.stringify(params, this.stringifyOptions)
+  get paramsSerializer(): ParamsSerializerOptions {
+    return {
+      serialize: (params) => qs.stringify(params, this.stringifyOptions),
+    }
```

Comment:
Might be able to use `indexes` instead of `serialize` + `qs` lib. Removing the need for `qs` altogether.

From the docs on axios Github:
```js
{
  // `paramsSerializer` is an optional config in charge of serializing `params`
  paramsSerializer: {
    encode?: (param: string): string => { /* Do custom ops here and return transformed string */ }, // custom encoder function; sends Key/Values in an iterative fashion
    serialize?: (params: Record<string, any>, options?: ParamsSerializerOptions ), // mimic pre 1.x behavior and send entire params object to a custom serializer func. Allows consumer to control how params are serialized.
    indexes: false // array indexes format (null - no brackets, false (default) - empty brackets, true - brackets with indexes)
  },
}
```